src/utils/source-search.js

```diff
@@ -58,8 +64,12 @@ function ignoreWhiteSpace(str) {
  * @memberof utils/source-search
  * @static
  */
-function searchOverlay(query) {
-  query = new RegExp(escapeRegExp(ignoreWhiteSpace(query)));
+function searchOverlay(query, regexMatch) {
+  if (regexMatch && query !== "") {
+    query = new RegExp(query);
```

Comment:
Of the top of my head , did you try `query = new RegExp(ignoreWhiteSpace(query));` ?